### ISSUE TITLE
Move official discussion from gitter

### ISSUE THREAD

**Issue by ivandotv** [mobxjs/mobx#2052]
3. [x] Idea:

It's hard to follow discussions on gitter or do any kind of meaningful search, maybe move to spectrum chat?
Also there is not much activity there so the switch should be relatively painless.

**Comment by FredyC** [comment]
Well, Gitter is definitely more like a quick chat, questions get repeated there for sure. Spectrum isn't exactly the best platform (yet), but having channels, topics & search is surely a win in terms of discussion organization.

There is already semi-official [community for mobx-state-tree](https://spectrum.chat/mobx-state-tree?tab=posts) with 460 members which is a fairly good number already. It would make sense to move there and add channels for different projects. @robinfehr would you be willing to share? :)

**Comment by ivandotv** [comment]
Any update on this? `mweststrate` gave the thumbs up. Let's do it.

**Comment by FredyC** [comment]
Well, @robinfehr did not respond about adopting the existing mobx-state-tree community so I guess it's mostly about creating a new one a letting people know about it. Feel free to start :)

**Comment by ivandotv** [comment]
Well there is a couple of things to do. 
1. Create the community on spectrum.chat.
2. Update the docs to point people to the new community.
3. Setup some kind of notification (a bot maybe) on gitter to tell people that the community has moved.

I have no authorization to do any of that :)

**Comment by FredyC** [comment]
1) Anyone can create a community at a spectrum
2) Anyone can submit a docs PR
3) Anyone can run such bot (if such thing exists)

**Comment by FredyC** [comment]
Spectrum community exists for a few weeks now, although with 20 members it doesn't have necessary traction yet. Perhaps over the time, it will improve.

https://spectrum.chat/mobx?tab=posts
